ui/builder.js

```diff
@@ -164,11 +164,16 @@ if (isE2E) {
 function buildHtml(inputFilename, outputFilename) {
   let result = fs.readFileSync(inputFilename).toString()
 
-  const placeholders = ['PUBLIC_URL']
+  let placeholders = ['PUBLIC_URL']
   placeholders.forEach((key) => {
     result = result.replace(new RegExp(`%${key}%`, 'g'), process.env[key])
   })
 
+  placeholders = ['TIME_PLACE_HOLDER']
```

Comment:
Why do we need to mutate the `placeholders` var? This seems to introduce an additional cost of understanding.

Reply:
How about just replacing the `TIME_PLACE_HOLDER` when we replace the time instead of all placeholders.

Reply:
Make sense, let me update.